config/config.go

```
@@ -857,21 +857,36 @@ func LogPelicanVersion() {
 	log.Infoln("Build Commit:", GetBuiltCommit())
 }
 
-// Print Pelican configuration to stderr
+// Dump Pelican configuration as structured logs and a human-readable format to stderr
 func PrintConfig() error {
 	rawConfig, err := param.UnmarshalConfig(viper.GetViper())
 	if err != nil {
 		return err
 	}
-	bytes, err := json.MarshalIndent(*rawConfig, "", "  ")
+
+	configBytes, err := json.MarshalIndent(rawConfig, "", "  ")
 	if err != nil {
 		return err
 	}
-	fmt.Fprintln(os.Stderr,
-		"================ Pelican Configuration ================\n",
-		string(bytes),
-		"\n",
-		"============= End of Pelican Configuration ============")
+
+	configDump := fmt.Sprintf(
+		"================ Pelican Configuration ================\n%s\n============= End of Pelican Configuration ============",
+		string(configBytes),
+	)
+
+	if param.Logging_LogLocation.GetString() != "" {
+		logFile, err := os.OpenFile(param.Logging_LogLocation.GetString(), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+		if err != nil {
+			return err
+		}
+		defer logFile.Close()
+
+		_, _ = logFile.WriteString(configDump + "\n")
```

What's the reason for ignoring the error output here?